sync/index.markdown

```diff
@@ -23,28 +23,28 @@ Below are the various GitHub repositories that are part of the Data-Sync feature
 
 ### Data-Sync Server
 
-Our Netty-based [Java Server](https://github.com/aerogear/aerogear-sync-server) exposes WebSocket and XMPP/GCM endpoint for our different clients.
+* [Java Sync Server Engine](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-core#aerogear-server-differential-synchronization-server-core) the Server Sync Engine.
+* [Netty Sync Server](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-netty#netty-differential-synchronization-server-implementation) which exposes WebSocket and XMPP/GCM endpoint for our different clients.
```

Comment:
I was not sure if we should or not. If you follow the link I think this is clear but it might not hurt pointing this out.